tools/free_energy/.shed.yml

```diff
@@ -0,0 +1,12 @@
+name: freeenergy
+owner: chemteam
+description: "Free energy tools of BRIDGE."
+remote_repository_url: https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/free_energy
+type: unrestricted
+categories:
+- Free Energy
+- Molecular Dynamics
+- Computational chemistry
+auto_tool_repositories:
```

Comment:
if it is, and ever will be, only one tool the suite/auto_tool_repository stuff is not needed.

Reply:
I think Tharindu wants to add some other tools

Reply:
@bgruening @simonbray Siomin is correct, I am planning to add more tools to setup and analyze free energy simulations.